docs/components/concepts/processes.md

```diff
@@ -4,82 +4,60 @@ title: "Processes"
 description: "Processes are flowchart-like blueprints that define the orchestration of tasks."
 ---
 
-Processes are flowchart-like blueprints that define the orchestration of **tasks**.
+A process is a defined sequence of distinct steps or tasks representing your business logic. Examples of a process could be an e-commerce shopping experience, or onboarding a new employee.
 
-For example, with Camunda you can [orchestrate human tasks](../../guides/getting-started-orchestrate-human-tasks.md).
+At large, process orchestration is a technology that coordinates the various moving parts (or endpoints) of a business process, and sometimes even ties multiple processes together. Process orchestration helps you work with the people, systems, and devices you already have—while achieving goals around end-to-end process automation.
 
-Every task represents a piece of business logic so the ordered execution produces a meaningful result.
+For example, with Camunda you can [orchestrate human tasks](../../guides/getting-started-orchestrate-human-tasks.md), [microservices](/guides/getting-started-orchestrate-microservices.mdl), and [APIs](/guides/getting-started-orchestrate-apis.md).
 
-A **[job worker](./job-workers.md)** implements the business logic required to complete a task. A job worker must be able to communicate with Camunda 8, but otherwise, there are no restrictions on its implementation. You can choose to write a worker as a microservice, but also as part of a classical 3-tier application, as a \(lambda\) function, via command line tools, etc.
+A **[job worker](./job-workers.md)** implements the business logic required to complete a task. You can choose to write a worker as a microservice, or also as part of a classical 3-tier application, as a \(lambda\) function, via command line tools, etc.
 
-Running a process requires three steps:
+Running a process broadly requires three steps:
 
 1. Deploy a process to Camunda 8.
 2. Implement and register job workers for tasks in the workflows.
-3. Create new instances of said process.
+3. Create new instances of the process.
 
 However, if you haven't yet, design the process:
 
-## BPMN 2.0
+## BPMN
 
-Zeebe uses [BPMN 2.0](http://www.bpmn.org/) to represent processes. BPMN is an industry standard widely supported by different vendors and implementations. Using BPMN ensures processes can be interchanged between Zeebe and other process systems.
+Camunda uses **[Business Process Model and Notation (BPMN)](/components/modeler/bpmn/bpmn.md)** to represent processes. The visual nature of BPMN enables greater collaboration between different teams, and is employed by numerous organizations globally.
```

Comment:
I'm not confident I know where BPMN is at these days with using or dropping the "2.0" 

Per the BPMN spec website:
> Will there be a major rewrite?
> Not for 2 or 3 years...